private-eye.js

```diff
@@ -6,11 +6,36 @@
     styles.innerHTML = 'a.private-link::after { content: "\\1F512"; font-size: 0.75em; vertical-align: top; }';
     document.body.appendChild(styles);
 
+    var DEFAULT_MESSAGE = "This is a link to a private site, which may or may not be accessible to you.";
+    var hrefValue;
+    var linkMessage;
```

Comment:
Sure, I've updated this. I was concerned with the performance of these variables being initialized on each iteration, since they're supposed to be defaults.